packages/esm-patient-banner-app/src/config-schema.ts

```diff
@@ -45,6 +45,45 @@ export const configSchema = {
         'Specifies the paper size for printing the sticker. You can define the size using units (e.g., mm, in) or named sizes (e.g., "148mm 210mm", "A1", "A2", "A4", "A5").',
       _default: 'A4',
     },
+    printMultipleStickers: {
+      enabled: {
+        _type: Type.Boolean,
+        _description: 'Whether to allow printing multiple patient ID stickers',
+      },
+      totalStickers: {
```

Comment:
The initial intention was to reduce the number of clicks for users that print multiple stickers since its disabled by default. 
<img width="822" alt="Screenshot 2024-11-12 at 10 31 08" src="https://github.com/user-attachments/assets/eade8c7d-bbec-4d3c-ae5b-16d9284954bf">

Reply:
Let me change that if its better

Reply:
In my latest commits, i have removed the `printMultipleStickers.enable` and i set the default toggle based on weather the total number of stickers to be printed is above 1.
`const [isMultipleStickersEnabled, setIsMultipleStickersEnabled] = useState(printMultipleStickers.numberOfStickers > 1); `